lib/ts/controllers/s-popover.ts

```diff
@@ -357,6 +371,14 @@ export class PopoverController extends BasePopoverController {
             cl.toggle(cls, show);
         });
     }
+
+    /**
+     * Toggles accessibility attributes based on whether the popover is shown or not
+     * @param {boolean=} show - A boolean indicating whether this is being triggered by a show or hide.
+     */
+    private toggleAccessibilityAttributes(show?: boolean) {
+        this.referenceElement.ariaExpanded = show?.toString() || this.referenceElement.ariaExpanded || 'false';
+    }
```

Comment:
Nice! This is all upside IMO.

Reply:
Would this apply to [tooltips](https://stackoverflow.design/product/components/popovers/#hover-tooltips) as well? If so, we'll want to move it into the `BasePopoverController` class

Reply:
Yes, I believe this should apply to tooltips too. Good thinking @b-kelly. Should be pretty trivial to move to `BasePopoverController`.

Reply:
Oh, I take it back. [From MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role):

> Though a tooltip may appear and disappear, as its appearance is automatic and not intentionally controlled by the user, the [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) role is not supported.

Reply:
Ok, in that case, let me change this to an approval. Thanks for the check on that!